src/main/java/kitEditor/KitEditor.java

```diff
@@ -1132,7 +1140,62 @@ private void trimAllSamples() {
                 "Done",
                 JOptionPane.INFORMATION_MESSAGE);
     }
+
+    private void duplicateSample(Sample sample) {
```

Comment:
I'm not sure I understood either what it does
but if it's like you describe, maybe it could be renamed to simply
`private void add(Sample sample);` ?

With some luck, then methods like `addSample(File wavFile)` could use it